### DISCUSSION TITLE
Unable to create record with an array that has multiple string

### DISCUSSION THREAD

**Discussion by jaywei1234** [pocketbase/pocketbase#2691] [Q&A] [answered]
![image](https://github.com/pocketbase/pocketbase/assets/56056463/de8d3f8a-4571-4c87-8a45-94d5dc4b1b35)

![image](https://github.com/pocketbase/pocketbase/assets/56056463/ce290d42-561c-4282-a206-22fecd1d4d45)

![image](https://github.com/pocketbase/pocketbase/assets/56056463/5786c2bf-d2bb-4061-b019-cc6c0110bade)

![image](https://github.com/pocketbase/pocketbase/assets/56056463/62cba7be-2684-46d4-8762-d47e0c2a5b13)
Above are the coding and the result of my project, I don't know what the problem is and can't solve it.

**Comment by ganigeorgiev** [comment, marked as answer]
When using [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData) the value must be a string or Blob/File. Array values can be added by calling `append` for each element. Or in other words you can try with:

```js
for (let item of tag) {
    formData.append("tag", item);
}
```

**Reply by jaywei1234** [reply]
Understood, it works now, thx a lot!!!

**Reply by ganigeorgiev** [reply]
_Side-note: based on the screenshot it seems that you are using an older PocketBase version (< 0.14.0) and I'd recommend to consider updating to the latest version to minimize eventual future breaking changes when jumping between major versions. As of now, there are no major breaking changes to the public web APIs when using the prebuilt executable and you should be to update by replacing the executable (make sure to have a backup of your `pb_data` just in case; for more details you can refer to [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md)._

**Reply by jaywei1234** [reply]
Got it, thank you so much!
